worldengine/drawing_functions.py

```diff
@@ -590,7 +588,8 @@ def _draw_a_mountain(pixels, x, y, w=3, h=3):
         pixels[x + modx, y + mody] = mcr        
 
 
-def _pseudo_random_land_pos(world, i):
+def _pseudo_random_land_pos(world, i):  # What is this good for? Why is it not truly random?
```

Comment:
I should remove my comment then.^^ Due to the recent changes to the RNG this function could technically be removed, I guess. It would require a couple of small changes on top of it, though.